PWGHF/TableProducer/candidateCreator2Prong.cxx

```diff
@@ -64,6 +74,8 @@ struct HfCandidateCreator2Prong {
   double bz = 0.;
 
   OutputObj<TH1F> hMass2{TH1F("hMass2", "2-prong candidates;inv. mass (#pi K) (GeV/#it{c}^{2});entries", 500, 0., 5.)};
+  OutputObj<TH2F> hKFMassFailed{TH2F("hKFMassFailed", ";inv. mass with topo fit failed;entries", 500, 0., 5., 3, -0.5, 2.5)}; // 0 for D0 failed, 1 for D0bar failed
+  OutputObj<TH1F> hMassTop2{TH1F("hMassTop2", "2-prong candidates;inv. mass (#pi K) (GeV/#it{c}^{2});entries", 600, -1.0, 5.)};
```

Comment:
Because when the topological constraint is failed, KF will return a negative or zero mass. I think the axis including a little negative value will help the users to notice this. But it could also be changed because we will not use the candidate mass after the topological constraint afterwards. this mass is only a reference to check whether some of the topological fittings are failed or not.